en/02_Developer_Guides/03_Forms/Field_types/04_GridField.md

```diff
@@ -242,17 +238,15 @@ $gridField->setConfig($config);
 
 ### `GridFieldConfig_RecordEditor`
 
-Similar to `GridFieldConfig_RecordViewer` with the addition support to edit or delete each of the records.
+> [!CAUTION]
+> Permission control for editing and deleting the record uses the `canEdit()` and `canDelete()` methods on the
+> `DataObject` object.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.